plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeDelete.java

```diff
@@ -101,7 +103,9 @@ public void testDeleteOssDeltaLakeMultiFile()
     private void testDeleteMultiFile(String tableName, String resourcePath)
     {
         hiveMinioDataLake.copyResources(resourcePath + "/lineitem", tableName);
-        getQueryRunner().execute(format("CREATE TABLE %s (dummy int) WITH (location = '%s')",
+        getQueryRunner().execute(format("CALL %s.system.register_table('%s', '%s', '%s')",
```

Comment:
nit, bring the first argument of the `format` method on the next line.

Same as here:

https://cs.github.com/trinodb/trino/blob/d4707c49c935f04177df669dc4a592d7c45d2546/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java#L2247-L2250